docs/retention-policy-agent/README.md

```diff
@@ -12,7 +12,75 @@ weight: 2
 The Results Retention Policy Agent removes older Results and their associated Records from the DB.
 
 ## Configuration
-The Results Retention Policy Agent config uses following field
 
-- `runAt`: determines when to run pruning job for DB.
-- `maxRetention`: for how long to keep results and records.
+The Results Retention Policy Agent is configured via the `tekton-results-config-results-retention-policy` ConfigMap.
+
+The following fields are supported:
+
+- `runAt`: Determines when to run the pruning job for the DB. It uses a cron schedule format. The default is `"7 7 * * 7"` (every Sunday at 7:07 AM).
+- `maxRetention`: The default retention period for how long to keep Results and Records when no specific policy matches. This can be a number (e.g., `30`), which is interpreted as days, or a duration string (e.g., `30d`, `24h`). The default is `30d`.
```

Comment:
`maxRetention` implies that it would also supersede any settings in `policies`. Is this correct? Do you mind clarifying how this behaves with specified retention policies?

Reply:
I will rewrite the doc to clarify that.

Reply:
I changed this to have it matched only Runs which aren't matched by any policy.  I can see that a feature request coming for that.  So let's do that instead now.